recipes/rdp-tools/meta.yaml

```diff
@@ -0,0 +1,37 @@
+
+package:
+  name: rdp-tools
+  version: "2.0.2"
+
+source:
+  git_url: https://github.com/rdpstaff/RDPTools.git
+  git_rev: 2.0.2
+
+requirements:
+  build:
+    - ant
+
+build:
+  number: 0
+  script:
+#    - brew update              # [osx]
+#    - brew install ant         # [osx]
+
+    - git config --global url."https://".insteadOf git://
+    - git submodule init
```

Comment:
I don't think this is needd if you use git_url. Recommended is a tarball if there is one packed with all submodules included.